README.md

```diff
@@ -22,7 +22,7 @@ Then exit and re-enter your shell.
 ```bash
 git clone https://github.com/osmosis-labs/osmosis
 cd osmosis
-git checkout v3.1.0
+git checkout v3.0.0
```

Comment:
I'm pretty sure `v3.0.0` is broken 😅 

cc @UnityChaos

Reply:
If we keep this at `v3.1.0` looks good to merge to me

Reply:
hmmmmm, ok so I had to actually go backwards to v2.0.0, let me check in on that sync

Reply:
Yup, v3.0.0 has this bug: https://github.com/cosmos/cosmos-sdk/issues/9800 because we forgot to init the capability module.